### PR TITLE
Add API to create slot

### LINKED ISSUE #22: API: Personen-IDs anlegen
HTTP-POST: /places/**place-id**/queues/**queue-id**
Payload JSON: { name: **string**}

Answer: 200/OK; JSON-Payload: { id: **string**; name: **string**; ticketNumber: **number**}
Answer ???/Not ok (not defined yet)

### PR DESCRIPTION
**Pull request by ohaz**
Closes #22

### CODE REVIEW

File: backend/digitales_warten.py [hunk 1]
```diff
@@ -99,17 +99,27 @@ def delete_queue(place_id, queue_id):
...
+        abort(404)
+    queue = Queue.query.filter_by(id=queue_id).filter_by(place=place).first()
+    if queue is None:
+        abort(404)
+    data = request.json
+    if 'name' not in data:
+        abort(400)
+    entry_name = data['name']
+    entry_id = generate_entry_id(entry_name)
+    ticket_number = Slot.query.filter_by(queue=queue).count() + 1
```
Comment (Gronner) [inline backend/digitales_warten.py]: This does not work, because removing users reduces count

Comment (ohaz) [reply backend/digitales_warten.py]: Done

File: backend/digitales_warten.py [hunk 2]
```diff
@@ -99,17 +99,28 @@ def delete_queue(place_id, queue_id):
...
+        abort(404)
+    queue = Queue.query.filter_by(id=queue_id).filter_by(place=place).first()
+    if queue is None:
+        abort(404)
+    data = request.json
+    if 'name' not in data:
+        abort(400)
+    entry_name = data['name']
+    entry_id = generate_entry_id(entry_name)
+    largest_previous_ticket_number = db.session.query(db.func.max(Slot.ticket_number)).scalar()
```
Comment (Gronner) [inline backend/digitales_warten.py]: Still buggy, very disappointet of you

Comment (ohaz) [reply backend/digitales_warten.py]: Next try

**Pull request merged**